containers/signing-service/clean-container.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+temporary_dependencies=(
+
+    gcc
+    libssl-dev
+    git
+    pkg-config
+    libsecp256k1-dev
+    libffi-dev
+    software-properties-common
+)
+
+# Clean up
+apt-get clean
+apt autoremove --assume-yes
+apt-get remove --purge --assume-yes ${temporary_dependencies[*]}
+
+rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Temporary file which is needed to build middleman protocol
+rm /usr/lib/RELEASE-VERSION
```

Comment:
This file can stay.  But it should be in a common subdirectory with everything else we install, not directly in `/usr/lib/`.

Reply:
Done.